content/version/1/4/code-of-conduct.de.md

```diff
@@ -39,7 +39,7 @@ Dieser Verhaltenskodex gilt sowohl innerhalb des Projektbereichs als auch in öf
 
 ## Umsetzung
 
-Fälle von missbräuchlichem, belästigendem oder anderweitig nicht akzeptablem Verhalten können dem Projektteam unter [EMAIL ADRESSE EINFÜGEN] gemeldet werden. Alle Beschwerden werden geprüft und untersucht und werden zu einer Antwort führen, die angesichts der Umstände für notwendig und angemessen gehalten wird. Das Projektteam ist verpflichtet, über diejenigen, die Vorfälle gemeldet haben, Verschwiegenheit zu wahren. Weitere Einzelheiten zu speziellen Umsetzungsgrundsätzen können gesondert mitgeteilt werden.
+Fälle von missbräuchlichem, belästigendem oder anderweitig nicht akzeptablem Verhalten können dem Projektteam unter [E-MAIL-ADRESSE EINFÜGEN] gemeldet werden. Alle Beschwerden werden geprüft und untersucht und werden zu einer Antwort führen, die angesichts der Umstände für notwendig und angemessen gehalten wird. Das Projektteam ist verpflichtet, über diejenigen, die Vorfälle gemeldet haben, Verschwiegenheit zu wahren. Weitere Einzelheiten zu speziellen Umsetzungsgrundsätzen können gesondert mitgeteilt werden.
```

Comment:
"Alle Beschwerden werden geprüft und untersucht"
 "prüfen" und "untersuchen" is almost the same and could be reduced to: "Alle Beschwerden werden geprüft und werden zu einer Antwort führen, die angesichts der Umstände für notwendig und angemessen gehalten wird"

Reply:
"Alle Beschwerden werden geprüft und untersucht" is closer to the literal english version though. 
See "All complaints will be reviewed and investigated [...]"

While I prefer the shorter version as well, I'm not sure how literal the translations should stay